src/views/Course/Course.js

```diff
@@ -87,6 +97,9 @@ function Course () {
             startDate={startDateResolver}
             endDate={endDateResolver}
             searchValue={searchValue}
+            setStartDate={setStartDate}
+            setEndDate={setEndDate}
+            setSearchValue={setSearchValue}
```

Comment:
Do you use these values in `Activity.js`? From what I can see they aren't being used.